src/aap_eda/utils/__init__.py

```diff
@@ -11,7 +11,33 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+from importlib.metadata import PackageNotFoundError, version
+from pathlib import Path
+
+import toml
 
 
 def str_to_bool(value: str) -> bool:
     return value.lower() in ("yes", "true", "1")
+
+
+def get_eda_version():
+    """Return EDA version as defined in the package or pyproject.toml."""
+    try:
+        return version("aap-eda")
+    except PackageNotFoundError:
+        return get_version_from_file()
```

Comment:
The logic of `get_version_from_file` is not reliable because it assumes that there is a repo or somehow a pyproject file without a package installed which is a weird scenario. 

I suggest to simplify the logic and rely only on importlib, which is the expected scenario, and returns a string like "package not installed properly" in case of `PackageNotFoundError`

In next iterations we can implement version in `aap_eda/__init__`, that should be a better fallback for `importlib.metadata.version` But I think there is no reason to support such a scenario where the package is not installed.